app/graphql/users/resolvers.js

```diff
@@ -0,0 +1,19 @@
+const { user: User } = require('../../models'),
+  logger = require('../../logger'),
+  errors = require('../../errors'),
+  { encryptPassword } = require('../../utils/users');
+
+exports.createNewUser = (parent, { user }) =>
+  encryptPassword(user.password)
+    .then(hash => {
+      user.password = hash;
+      return User.createModel(user);
```

Comment:
you could just do
`.then(hash => User.createModel({ ...user, password: hash })`
to keep it in a single line